data/json/items/gun/22.json

```
@@ -118,7 +118,7 @@
     "copy-from": "ruger_lcr_38",
     "type": "GUN",
     "name": { "str_sp": "Ruger LCR .22" },
-    "weight": "420 g",
+    "weight": "400 g",
```

As per discussion between Scarf and Robbie, reversion of gun weights since already at nearest 10 grams
```suggestion
    "weight": "420 g",
```